tests/testsuite/freshness.rs

```diff
@@ -2493,6 +2493,9 @@ LLVM version: 9.0
             "src/main.rs",
             r#"
             fn main() {
+                if std::env::args_os().any(|s| s.into_string().unwrap().contains("unstable-options")) {
```

Comment:
Why do we need this guard? Seems the nightly channel detection is off?

Reply:
This test case was failing for me locally, and this fixed the issue. However i'm not seeing this fail now, so it's possible it can be reverted.